ext/pdo/pdo_stmt.c

```diff
@@ -1255,7 +1229,7 @@ PHP_METHOD(PDOStatement, fetchAll)
 			do_fetch_class_prepare(stmt);
 			break;
 
-		case PDO_FETCH_FUNC: /* Cannot be a default fetch mode */
+		case PDO_FETCH_FUNC: /* Cannot be a default fetch mode */ {
```

Comment:
I'm not a fan of braces in switch. Couldn't this be done in some other way? A dedicated function wasn't so bad before.

Reply:
made a function again :)